examples/experimental/onnx_ptq_classification.py

```diff
@@ -0,0 +1,71 @@
+import os
+
+import torchvision
+from torchvision import transforms
+
+import onnx
+
+from nncf.experimental.post_training.api.dataloader import DataLoader
+from nncf.experimental.post_training.compressed_model import CompressedModel
+from nncf.experimental.post_training.compression_builder import CompressionBuilder
+from nncf.experimental.post_training.quantization.algorithm import PostTrainingQuantization
+from nncf.experimental.post_training.utils import export
+
+from nncf.experimental.post_training.quantization.algorithm import PostTrainingQuantizationParameters
+from nncf.experimental.post_training.initialization.algorithm import InitializationAlgorithms
+from nncf.experimental.post_training.initialization.quantizer_range_finder import QuantizerRangeFinderParameters
+
+
+def create_train_dataset(dataset_dir):
```

Comment:
Not sure that we should define the method using the `train` in the naming. Post-training quantization doesn't require model re-training.